test/utils/http_static_test_server.py

```diff
@@ -28,6 +29,7 @@ class HTTPStaticTestServer(object):
     """
 
     def __init__(self, port=8088):
+        HTTPServer.address_family = socket.AF_INET6
```

Comment:
Here's a brief snippet that demonstrates problem:
https://paste.fedoraproject.org/300630/50106110/

After overwriting `Foo.x`, not only is the behavior of the first `Foo` instance changed, any instantiations of `Foo` after the attribute is change are also affected by the change.

In this case, the method using the overridden method (`self.address_family`) is `__init__`, so changing `HttpServer.address_family` means that every instance created after this change has the new IPv6 behavior, since instances created before it have already run their `__init__` method.

Since this is a potentially backward-incompatible change due to poor IPv6 implementations, it's safer to override the `address_family` class attr in a subclass, as recommended in the link in my previous comment.